lib/Interpreter/CMakeLists.txt

```diff
@@ -124,7 +104,6 @@ else()
       clangStaticAnalyzerCore
       )
   endif(LLVM_LINK_LLVM_DYLIB)
-
   add_llvm_library(clangCppInterOp
     DISABLE_LLVM_LINK_LLVM_DYLIB
```

Comment:
Not sure what role this plays for plays for the emscripten build. I remember as we were forcing a shared build ... we were using 
```
  add_llvm_library(clangCppInterOp
    SHARED
    ......

```

Reply:
Just wanted to make sure `DISABLE_LLVM_LINK_LLVM_DYLIB` plays a part for the emscripten build as we weren't using it earlier. If you're confident we need it, i'm happy